src/NuGet.Packaging.Build/project.json

```diff
@@ -1,11 +1,11 @@
 ﻿{
-    "version": "3.1.2-*",
+    "version": "3.2-*",
     "preprocess": "../../compiler/preprocess/*.cs",
     "dependencies": {
-        "NuGet.Versioning": "3.1.2-*",
-        "NuGet.Packaging": "3.1.2-*",
+        "NuGet.Versioning": "3.2-*",
+        "NuGet.Packaging": "3.2-*",
         "Newtonsoft.Json": "6.0.6",
```

Comment:
Not sure about that. @emgarten @deepakaravindr, does any one of you know the answer?